rules/aip0158/pagination.go

```diff
@@ -0,0 +1,113 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 		https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package aip0158
+
+import (
+	"fmt"
+
+	"github.com/googleapis/api-linter/lint"
+	"github.com/jhump/protoreflect/desc"
+	"github.com/jhump/protoreflect/desc/builder"
+)
+
+var requestPaginationPageSize = &lint.MessageRule{
+	Name: lint.NewRuleName("core", "0158", "request-message", "page_size-field"),
+	URI:  "https://aip.dev/158",
+	LintMessage: func(m *desc.MessageDescriptor) (problems []lint.Problem) {
+		// We only care about Get methods for the purpose of this rule;
+		// ignore everything else.
+		if !isListRequestMessage(m) {
```

Comment:
Nit: Now that #190 is in, it would be awesome if this could use the `OnlyIf` syntax. Applies throughout the file.

This does _not_ block a merge.

Reply:
I just went ahead and did this.